addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -18,15 +18,18 @@
 <!-- PAGE INDEX -->
 <!-- ============================================================ -->
 <template id="forum_model_nav">
+    <t t-set="_forum_slug" t-value="slug(forum) if forum else 'all'"/>
+    <t t-set="_all_forums"><t>All forums</t></t>
```

Comment:
Extra "t" is not necessary normally, is it ?

Reply:
You are right. Extra "t" is useless. Removed.